### ISSUE TITLE
Missing files in sdist

### ISSUE THREAD

**Issue by thatch** [wbond/asn1crypto#183]
It appears that the manifest is missing at least one file necessary to build
from the sdist for version 1.3.0. You're in good company, about 5% of other
projects updated in the last year are also missing files.

```
+ /tmp/venv/bin/pip3 wheel --no-binary asn1crypto-tests -w /tmp/ext asn1crypto-tests==1.3.0
Looking in indexes: http://10.10.0.139:9191/root/pypi/+simple/
Collecting asn1crypto-tests==1.3.0
  Downloading http://10.10.0.139:9191/root/pypi/%2Bf/a3d/10b50ea171f37/asn1crypto_tests-1.3.0.tar.gz (547 kB)
    ERROR: Command errored out with exit status 1:
     command: /tmp/venv/bin/python3 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-64_gpyyj/asn1crypto-tests/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-64_gpyyj/asn1crypto-tests/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-wheel-64_gpyyj/asn1crypto-tests/pip-egg-info
         cwd: /tmp/pip-wheel-64_gpyyj/asn1crypto-tests/
    Complete output (2 lines):
    running egg_info
    error: [Errno 2] No such file or directory: '/tmp/pip-wheel-64_gpyyj/LICENSE'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

**Comment by wbond** [comment]
Thanks for letting me know!

**Comment by wbond** [comment]
Hmm, the file at https://files.pythonhosted.org/packages/f4/90/8ff25249b85389ce17b4922dccf6acbf5727756e896befde5c77ffcf9684/asn1crypto_tests-1.3.0.tar.gz definitely has a `LICENSE` file in it.

It would appear maybe this is a bug in the `wheel` command of pip?

**Comment by wbond** [comment]
No only is there a `LICENSE` file in the root of the `.tar.gz`, but there is a `LICENSE` file in the `ans1crypto_tests.egg-info/` dir.

**Comment by thatch** [comment]
It looks like it has to do with PACKAGE_ROOT trying to refer outside of the sdist?  I'm not 100% sure, the setup.py is pretty custom.  This works for the main asn1crypto package, just fails for asn1crypto-tests.  The script in the PR is slightly different, and there may be a second bug around sdist with flags.

**Comment by wbond** [comment]
I think you are likely right - probably worked for me since I build wheels and sdist from the git repo, whereas the wheel command is downloading the sdist and then running `egg_info`.

**Comment by wbond** [comment]
https://github.com/wbond/asn1crypto/commit/116f2e5eb6d4c989418f57b0b3748ce0edfc1554 should have a fix for this issue. If you have a couple of minutes to test, I'd be grateful!

**Comment by wbond** [comment]
This should be fixed in the 1.4.0 release
